packages/cdai/package.json

```diff
@@ -12,6 +12,31 @@
     "cdai"
   ],
   "scripts": {
-    "storybook": "start-storybook -p 6006"
+    "storybook": "start-storybook -p 6006",
+    "build": "node scripts/build.js"
+  },
+  "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
```

Comment:
Should these be `devDependencies` versus dependencies, or will they be included at runtime?

Reply:
We're having this discussion/debate in the CDAI guild :)
Currently they are in the same place as in the 'ide-components' repo.
The argument is that such plugins insert code which is executed at runtime, therefore that should count as a dependency instead of devDependency. Some security scanning tools ignore devDependencies, so this approach is safer, as a vulnerability in such a plugin could potentially affect the running application.